packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerSubItemAnimatedExpandableContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { useExpandedAnimation } from '@/settings/hooks/useExpandedAnimation';
+import styled from '@emotion/styled';
+import { AnimatePresence, motion } from 'framer-motion';
+import { ReactNode } from 'react';
+
+const StyledAnimatedContainer = styled(motion.div)`
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  width: 100%;
+`;
+
+type NavigationDrawerSubItemAnimatedExpandableContainerProps = {
+  children: ReactNode;
+  isOpen?: boolean;
+};
+
+export const NavigationDrawerSubItemAnimatedExpandableContainer = ({
```

Comment:
I rather put this component in something more generic as it's not linked to any right drawer core logic, and rename it in something like `AnimatedExpandable`

Reply:
yep! agreed
https://github.com/twentyhq/twenty/pull/9287#discussion_r1899993792